pages/styles/styles.css

```
@@ -0,0 +1,213 @@
+* {
+    font-family:Helvetica;
+}
+body {
+    background-color: pink;
+}
+/********** about grid container*************************** */
+.container-about {
+    height: 100vh;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+}
+.cell {
+    color: white;
+    font-size: 1rem;
+    font-style: bold;
+    text-align: center;
+    padding: 2rem;
+}
+.cell-1 {
+    grid-row: 1;
+    grid-column: 1/4;
+    padding: 3rem;
+}
+.cell-2 {
+    grid-row: 2/3;
+    grid-column: 1/2;
+    padding: 0.5rem;
+    align-content: center;
+    margin-top: 4rem;
+}
+.cell-3 {
+    grid-row: 2/3;
+    grid-column: 2/4;
+    font-size: 1.5rem;
+    text-align: justify;
+    padding: 0.5rem;
+    margin-top: 1.5rem;
+}
+.cell-4 {
+    grid-row: 3/4;
+    grid-column: 2/4;
+    padding: .5rem;
+}
+.cell-5 {
+    grid-row: 4/4;
+    grid-column: 2/3;
+}
+/********* navigation bar (header) ***************/
+nav ul {
+    display:flex;
+    background-color: white;
+    list-style-type: none;
+    position: fixed;
+    top: 0;
+    width: 100%;
+}
+li a {
+    display:block;
+    color:rgb(162, 221, 201);
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-weight: bold;
+    font-size: 20pt;
+}
+li a:hover {
+    background-color: grey;
+}
+nav li {
+    display:block;
+    padding:8px;
+    list-style-type: none;
+}
+
+img{
+    max-width: 100%;
+    height: 50vh;
+}
+
+/* ***********footer (contact)******************* */
+.vertical-footer {
+    margin-bottom: 0%;
+    list-style-type: none;
+    width: 500px;
+    align-content:right;
```

`right` is not a value here

```suggestion
    align-content: flex-end;
```